docs/blog/abac-repo-permissions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: ACR ABAC Repository Permissions Private Preview Blog Announcement
+description: Private preview blog announcement for the ACR ABAC Repository Permissions feature.
+ms.topic: post
+ms.date: 07/15/2024
+ms.author: johsh
+author: johnsonshi
+ms.custom:
+---
+
+> [!NOTE]
```

Comment:
[!NOTE] The ACR ABAC Repository Permissions feature is currently in private preview. For details on enrolling in the Private Preview and to ensure a smooth experience, please follow the provided instructions.